### PR TITLE
new transfer added

### PR DESCRIPTION
**Pull request by snowflake-dev918**

### CODE REVIEW

File: contracts/Crea2orsNFT_v3.3.sol [hunk 1]
```diff
@@ -7,12 +7,19 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 import "@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol";
 
+interface ICrea2Crypto {
+  function transfer(address to, uint tokens) external;
+  function balanceOf(address addr) external returns (uint256);
+  function transferFrom(address from , address to, uint tokens) external;
+}
+
```
Comment (devlancer412) [inline contracts/Crea2orsNFT_v3.3.sol:15]: Add some comment here

File: contracts/Crea2orsNFT_v3.3.sol [hunk 2]
```diff
@@ -45,12 +53,14 @@ contract Crea2orsNFT is ERC1155, Ownable, EIP712 {
         string memory name_,
```
Comment (devlancer412) [inline contracts/Crea2orsNFT_v3.3.sol:53]: Don't use memory variable

File: contracts/Crea2orsNFT_v3.3.sol (hunk 1 above)
Comment (snowflake-dev918) [reply contracts/Crea2orsNFT_v3.3.sol:15]: Thanks for your help.
I checked.

File: contracts/Crea2orsNFT_v3.3.sol (hunk 2 above)
Comment (snowflake-dev918) [reply contracts/Crea2orsNFT_v3.3.sol:53]: ok I see

**Pull request merged**